Modules/Microsoft365DSC/DSCResources/MSFT_AADRoleSetting/MSFT_AADRoleSetting.psm1

```diff
@@ -0,0 +1,1232 @@
+        [System.Boolean]
+        $EligibleApproveNotificationOnlyCritical,
+
+        [Parameter()]
+        [System.Boolean]
+        $ActiveAlertNotificationDefaultRecipient,
+
+        [Parameter()]
+        [System.String[]]
+        $ActiveAlertNotificationAdditionalRecipient,
+
+        [Parameter()]
+        [System.Boolean]
+        $ActiveAlertNotificationOnlyCritical,
+
+        [Parameter()]
+        [System.Boolean]
+        $ActiveAssigneeNotificationDefaultRecipient,
+
+        [Parameter()]
+        [System.String[]]
+        $ActiveAssigneeNotificationAdditionalRecipient,
+
+        [Parameter()]
+        [System.Boolean]
+        $ActiveAssigneeNotificationOnlyCritical,
+
+        [Parameter()]
+        [System.Boolean]
+        $ActiveApproveNotificationDefaultRecipient,
+
+        [Parameter()]
+        [System.String[]]
+        $ActiveApproveNotificationAdditionalRecipient,
+
+        [Parameter()]
+        [System.Boolean]
+        $ActiveApproveNotificationOnlyCritical,
+
+        [Parameter()]
+        [System.Boolean]
+        $EligibleAssignmentAlertNotificationDefaultRecipient,
+
+        [Parameter()]
+        [System.String[]]
+        $EligibleAssignmentAlertNotificationAdditionalRecipient,
+
+        [Parameter()]
+        [System.Boolean]
+        $EligibleAssignmentAlertNotificationOnlyCritical,
+
+        [Parameter()]
+        [System.Boolean]
+        $EligibleAssignmentAssigneeNotificationDefaultRecipient,
+
+        [Parameter()]
+        [System.String[]]
+        $EligibleAssignmentAssigneeNotificationAdditionalRecipient,
+
+        [Parameter()]
+        [System.Boolean]
+        $EligibleAssignmentAssigneeNotificationOnlyCritical,
+
+        [Parameter()]
+        [System.Management.Automation.PSCredential]
+        $Credential,
+
+        [Parameter()]
+        [System.String]
+        $ApplicationId,
+
+        [Parameter()]
+        [System.String]
+        $TenantId,
+
+        [Parameter()]
+        [System.String]
+        $ApplicationSecret,
+
+        [Parameter()]
+        [System.String]
+        $CertificateThumbprint
+    )
+
+    Write-Verbose -Message "Getting configuration of Role"
+    $ConnectionMode = New-M365DSCConnection -Workload 'MicrosoftGraph' `
+        -InboundParameters $PSBoundParameters `
+        -ProfileName 'beta'
+
+    Select-MgProfile -Name 'beta'
+    $MaximumFunctionCount = 32000
+
+    Write-Verbose -Message "Getting configuration of Role"
+
+    #Ensure the proper dependencies are installed in the current environment.
+    Confirm-M365DSCDependencies
+
+    #region Telemetry
+    $ResourceName = $MyInvocation.MyCommand.ModuleName -replace "MSFT_", ""
+    $CommandName  = $MyInvocation.MyCommand
+    $data = Format-M365DSCTelemetryParameters -ResourceName $ResourceName `
+        -CommandName $CommandName `
+        -Parameters $PSBoundParameters
+    Add-M365DSCTelemetryEvent -Data $data
+    #endregion
+
+    $nullReturn = $null
+
+    function getUserUPN{
```

Comment:
Please use the code styling standards:
- Place the open bracket on a new line. This issue exists on multiple places in this file, like lines 214, 239, 275 (NOTE: don't do it with the Where-Object parameter, it doesn't work there)
- Use the Verb-Noun format for this function: Get-UserUPN